src/web/package.json

```diff
@@ -27,6 +27,7 @@
     "yup": "0.32.11"
   },
   "devDependencies": {
+    "@senecacdot/eslint-config-telescope": "workspace:*",
```

Comment:
Fascinating, how does this work?  I've never seen this before.

Reply:
basically, `eslintrc.js` will reuse the module `@senecadot/eslint-config-telescope` each time, and the `workspace:*` means that it will look inside the current workspace instead of searching in the npmjs registry.

Reply:
I wonder if this holds the secret for moving Satellite into the repo?  If there was a way to make this resolution work within a Dockerfile, we'd be set.

Reply:
Yes, this is how I was doing it for Satellite but I ran into a wall when I couldn't figure out Docker.